myStyle.css

```diff
@@ -0,0 +1,38 @@
+  }
+div p {
+    color: navy;
+    margin-left: 20px;
+  }
+
+#ID {
+    background-color: red;
+  }
+
+.nice {
+    font-family: sans-serif;
+  }
+
+[target=_blank] {
+    background-color: yellow;
+  }
+
+p[target=_hi] {
+    background-color: rgb(212, 0, 255);
+  }
+
+div a{
+    background-color: lightgrey;
+    width: 300px;
+    border: 5px solid green;
+    padding: 2px;
+    margin: 150px;
+    line-height: 100px;
+    }
+.ex{
```

Comment:
it's good to be consistent with spacing, so
```
.ex {
  ...
}
```
is preferred and consistent with the code above.
Rather than:
```
.ex{
  ...
}
```

In a normal project, the linter would automatically catch this for us